boefjes/boefjes/katalogus/local_repository.py

```diff
@@ -62,20 +62,25 @@ def schema(self, id_: str) -> dict | None:
 
     def cover_path(self, id_: str) -> Path:
         boefjes = self.resolve_boefjes()
+        normalizers = self.resolve_normalizers()
+        default_cover_path = self.default_cover_path()
 
-        if id_ not in boefjes:
-            return self.default_cover_path()
-
-        boefje = boefjes[id_]
-        path = boefje.path / "cover.jpg"
+        if id_ not in boefjes and id_ not in normalizers:
+            return default_cover_path
```

Comment:
You might as well move this to an else-clause in the if-clause below

Reply:
I have removed this as well, will be indeed handled in the else clasue